net/modemmanager/files/modemmanager.proto

```diff
@@ -402,6 +411,45 @@ proto_modemmanager_setup() {
 		proto_block_restart "${interface}"
 		return 1
 	}
+	
+	# check if Signal refresh rate is set
+	if [ -n "${signalrate}" ] && [ "${signalrate}" -eq "${signalrate}" ] 2>/dev/null; then
+		echo "setting signal refresh rate to ${signalrate} seconds"
+		mmcli --modem="${device}" --signal-setup="${signalrate}"
+		
+		# log signal information
+		local modemsignal
+		modemsignal=$(mmcli --modem="${device}" --signal-get --output-keyvalue)
+		
+		cdma1x_rssi=$(modemmanager_get_field "${modemsignal}" "modem.signal.cdma1x.rssi")
```

Comment:
I would not do this. There's no point in printing a snapshot of signal information in the log as soon as the extended signal support is enabled. Having an option to enable extended signal support would be fine, but this print is a bit useless IMO :)